test/test_performance.py.in

```diff
@@ -82,7 +98,7 @@ def generate_test_description():
     return LaunchDescription([
         node_under_test,
         launch_testing.util.KeepAliveProc(),
-        launch_testing.actions.ReadyToTest(),
+        launch.actions.OpaqueFunction(function=lambda context: ready_fn()),
```

Comment:
Is this the only real difference? Maybe we should just use this API so we don't need to create a separate branch right now. Thoughts?

Reply:
Maybe we can use the same API with Eloquent. I will try it

Reply:
Sweet. If it works, let's just propose this line change for the `master` branch and delete the `dashing` one.